lib/screens/profile_screen.dart

```diff
@@ -18,12 +18,14 @@ class ProfileScreen extends StatefulWidget {
 
 class ProfileScreenState extends State<ProfileScreen> {
   String preferredSourceLanguage = "";
+  String preferredTargetLanguage = "";
 
   @override
   void initState() {
-    _getPreferredSourceLanguage().then((value) {
+    _getPreferredLanguage().then((values) {
       setState(() {
-        preferredSourceLanguage = value ?? defaultSourceLang;
+        preferredSourceLanguage = values[0] ?? defaultSourceLang;
+        preferredTargetLanguage = values[1] ?? "";
```

Comment:
It seems we should have a default target languages as we do for source lang

Reply:
Okay. Will add it in the default configuration